atlassian/confluence.py

```diff
@@ -712,9 +712,14 @@ def is_page_content_is_already_updated(self, page_id, body, title=None):
         :param title: Title to compare
         :return: True if the same
         """
-        confluence_content = (((self.get_page_by_id(page_id, expand='body.storage') or {})
-                               .get('body') or {})
-                              .get('storage') or {})
+        if self.advanced_mode:
+            confluence_content = (((self.get_page_by_id(page_id, expand='body.storage').json() or {})
```

Comment:
Hi!
maybe better will be something like that:
```py
page = self.get_page_by_id(page_id, expand='body.storage') 
page = page.json() if self.advanced_mode else page
confluence_content = (((page or {})
              .get('body') or {})
              .get('storage') or {})
```
it helps to avoid code duplication and makes code more readable.